WordPress/Tests/Security/NonceVerificationUnitTest.1.inc

```diff
@@ -524,3 +524,110 @@ function different_function_name() {
     update_post_meta( (int) $_POST['id'], 'a_key', $_POST['a_value'] );
 }
 // phpcs:set WordPress.Security.NonceVerification customNonceVerificationFunctions[]
+
+function test_fully_qualified_call_to_global_nonce_verification_function() {
+    if ( ! IS_NUMERIC( $_POST['foo'] ) ) {
+        return;
+    }
+
+    \wp_verify_nonce( 'some_action' ); // OK.
```

Comment:
```suggestion
    namespace\wp_verify_nonce( 'some_action' );
    \wp_verify_nonce( 'some_action' );
```

Note: not sure if these can be combined in one test case snippet or if we need a secondary test function for this. Please check. If you want to play it safe, just have a second function, then it should be clear when namespace tracking gets implemented that the expectation for that particular test case changes.